components/shared_code/src/shared_data_model/parameters.py

```diff
@@ -151,6 +151,15 @@ class Branch(StringParameter):
     metrics: list[str] = ["source_up_to_dateness"]
 
 
+class Branches(MultipleChoiceWithAdditionParameter):
+    """Branches parameter."""
```

Comment:
Is the plan to include this parameter in the other sources (jenkins, sonar, devops, etc) as well?